ros/src/common/libs/diagnostics_lib/diag_msgs/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <name>diag_msgs</name>
+  <version>1.6.3</version>
+  <description>The libdiag package</description>
+
+  <maintainer email="yusuke.fujii@tier4.jp">Yusuke FUJII</maintainer>
```

Comment:
please change your name and your mail

Reply:
remove fujii san's name from maintainer.